### ISSUE TITLE
Thoughts on WPRig child themes

### ISSUE THREAD

**Issue by paaljoachim** [wprig/wprig#282]
## Issue Overview
Well I have watched some of the WPRig videos and became very fascinated by the setup.

I am using getbeans.io for the moment https://hellofromtonya.com/blog/met-beans-theme-framework/ I even made a tutorial site for it. What I like with the Beans Framework is that I can add UiKit modifications to it. It feels very lean and fast. It just feels easy to customize. Beans also uses a parent to "house" most of the files and the child theme has less files and code to relate to.

I am not much of a coder, but I am good at finding resources to learn from and use it in the child themes that I am using. That would for instance mean finding various code snippets, how to guides and references.

Bottom line for me is this: Having a simple child theme with very few files to relate to. Yes I can create a few CSS and PHP files, but having the more complex options in the parent and having a much easier to handle lighter coded child theme. That would mean I would just download the most optimized WPRig parent theme with the most important connections to the child. Find some child themes I like the layout of and then download the one that comes closest to the look I want. Then begin to customize it. Without having to think of Gulp, NPM etc etc.

**Comment by mor10** [comment]
I'm working on documentation around child themes. The short version: You can build a child theme off any theme generated using WP Rig. You do not use WP Rig itself to build a child theme.

**Comment by paaljoachim** [comment]
Hei Morten..:)

It would be good if atleast one starter/beginner child theme + various code snippets with comments were added to the functions file. It would be a good way to learn how to use WPRig as a parent and the starter child theme with some code snippets. This way I could begin to experiment with the child theme, learn from the code snippets and begin to create various designs.

It would likely be good for devs to be a part of modifying the WPRig theme. Then for others use child themes. That could mean having one parent theme and multiple child themes that have their own functions and designs. Learning from how various child themes are put together and the code snippets they use. I learnt a lot from picking up code from the beans support forum. https://community.getbeans.io/forum/support/
I have made lots of child themes for various clients and also built a tutorial site: https://wpbeansframework.com. The Beans framework has felt like a ghost town for a while I am slowly beginning to think that I should perhaps begin to use WPRig because of the development that is going on with it.

**Comment by mor10** [comment]
Related to #270

**Comment by jacklowrie** [comment]
This is discussed further (with explanations and alternate workflows) on #260,  #270, and #399.
